public/js/visualizer.js

```diff
@@ -350,18 +350,15 @@ function toLatLng (point){
 // ex: getKeyMapping("Fallout New Vegas", "Attacked")
 var getKeyMapping = function(game, eventName){
 
-  // Get the type of event from the lookup table
-  var type = options.lookup_table[eventName];
-
-  // Find mapping
-  var mapping = _.findWhere(options.mappings, {game : settings.game, type : type});
-
-  if (!mapping || !type) {
-    console.error("Unable to find key mapping for: " + eventName);
-    return;
+  for (index in options.mappings) {
```

Comment:
This works. Going forward, I'd usually suggest the "_.findWhere()" function for the sake of brevity. 

Regardless, I like that the actions field of key mappings is only ever accessed from a single place.